src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -634,7 +656,7 @@ var SimpleEventPlugin = {
   },
 
   willDeleteListener: function(inst, registrationName) {
-    if (registrationName === 'onClick') {
+    if (registrationName === 'onClick' && isInteractive(inst._tag)) {
```

Comment:
Isn't this the opposite of what we want? i.e., we want to add the empty click handler to elements that aren't considered interactive?